src/content/app/species/components/species-title-area/species-remove/SpeciesRemove.scss

```diff
@@ -40,3 +40,8 @@ $fake-left-margin: 60px;
   color: $blue;
   cursor: pointer;
 }
+
+.buttonContainer {
+  display: flex;
+  justify-content: flex-end;
+}
```

Comment:
If you are also modifying the species removal, notice how the XD says "Remove species" instead of "Remove"?

![image](https://user-images.githubusercontent.com/6834224/182559352-7022ce68-fbda-4f99-a942-1895a7061280.png)

and how it also keeps the "do not remove" flush right?

![image](https://user-images.githubusercontent.com/6834224/182559414-690d3c35-97d5-4285-a1d1-e571adaf993f.png)

Reply:
Good spot. Updated the text

Reply:
You aren't aligning it to the right corner?


https://user-images.githubusercontent.com/6834224/182611039-298b05a3-ea77-4641-919a-cbbf157b8c7b.mov

Reply:
sorry miss that, have now fix it.